src/main/docker/docker-compose.yml

```diff
@@ -6,16 +6,226 @@ services:
         build:
             context: ../../..
             dockerfile: src/main/docker/Dockerfile
-        volumes:
-            - ../resources/config/application-dev.yml:/opt/artemis/application-dev.yml:ro
-            - ../resources/config/application-artemis.yml:/opt/artemis/application-artemis.yml:ro
+        env_file: .env
         environment:
-            _JAVA_OPTIONS: -Xmx5120m -Xms2560m
-            SPRING_PROFILES_ACTIVE: dev,bamboo,bitbucket,jira,artemis,scheduling,athene,local
-            SPRING_DATASOURCE_URL: jdbc:mysql://artemis-mysql:3306/Artemis?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC
-            SPRING_DATASOURCE_USERNAME: root
-            SPRING_DATASOURCE_PASSWORD:
-            JHIPSTER_SLEEP: 30 # gives time for other services to boot before the application
+            # ===================================================================
+            # Artemis specific properties
+            # ===================================================================
+
+            artemis.course-archives-path: ./exports/courses         # a folder in which archived courses and exams are stored.
+            artemis.repo-clone-path: ./repos                        # a folder in which git repos for the online code editor are stored. In a multi node setup, this folder should be in a shared file system area (e.g. based on NFS), so that user can access the same files over multiple nodes
```

Comment:
According to [this](https://docs.artemis.ase.in.tum.de/dev/setup/?highlight=docker%20compose#run-the-server-via-docker) doc, the path should rather be `./data/repos` so there are no permission errors.
Same for `artemis.repo-download-clone-path`, `artemis.course-archives-path`, `artemis.submission-export-path`, and `artemis.file-upload-path`. This was an issue for me when I first tried to setup Artemis via docker

Reply:
Hi @TheZoker , I could not find where it suggests being ./data/repos. Unfortunately I only found the reference in the server part in which says to be ./repos. In the docker part it says that it will use the folder  /opt/artemis/data to install any data. Could you please point out which part specifically says that?

Reply:
Lets handle volumes in a next small pr ;)

Reply:
This is handled in #5624.